app/ui/src/main/java/com/fsck/k9/ui/settings/account/AccountSettingsFragment.kt

```diff
@@ -315,7 +325,11 @@ class AccountSettingsFragment : PreferenceFragmentCompat() {
         private const val PREFERENCE_SENT_FOLDER = "sent_folder"
         private const val PREFERENCE_SPAM_FOLDER = "spam_folder"
         private const val PREFERENCE_TRASH_FOLDER = "trash_folder"
+        private const val PREFERENCE_OPEN_NOTIFICATION_SETTINGS = "open_notification_settings"
         private const val DELETE_POLICY_MARK_AS_READ = "MARK_AS_READ"
+        private val NOTIFICATION_PREFERENCES = arrayOf("account_ringtone", "account_vibrate",
```

Comment:
These are not all notification preferences. The name should better reflect what it's used for. Maybe `PRE_SDK26_NOTIFICATION_PREFERENCES`?

Reply:
Good point. Changed to your suggestion.